pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -342,7 +342,7 @@ void run() throws PulsarAdminException {
         private String retentionTimeStr;
 
         @Parameter(names = { "--size", "-s" }, description = "Retention size limit (eg: 10M, 16G, 3T). "
-                + "0 means no retention and -1 means infinite size retention", required = true)
+                + "0 or less than 1MB means no retention and -1 means infinite size retention", required = true)
```

Comment:
This is actually very confusing. Why can't i specify 1kb?

Reply:
because that is the current logic. the broker only takes MB as inputs. so if you specify less than 1MB, it is 0. This change here is just to update the documentation. we can discuss if we want to change this behavior, which will be a bunch of BC considerations.